vhost-device-scmi/src/devices/iio.rs

```diff
@@ -159,6 +159,8 @@ const UNIT_MAPPING: &[UnitMapping] = &[
     },
 ];
 
+const IIO_DEFAULT_NAME: &str = "iio";
```

Comment:
Should we document this default in the help page and README?

Reply:
Ok, I will document it in help page and README in next push

Reply:
If a user of vhost-device-scmi sets name with command parameter, it will use his setting. If not, it will use name from sysfs name attribute, otherwise, it will use this default name.

In my opinion, I think we can skip this, it will not influcene user. It is a guarantee for scmidevice to have a name.